### DISCUSSION TITLE
Using SG relays with Home assistant

### DISCUSSION THREAD

**Discussion by ap20132022** [raomin/ESPAltherma#258] [Q&A]
Hi,
I have bought some relays to control and control my heat pump.
The switch.altherma works fine.
How can you put the SG Signal (espaltherma/sg/set) to Home assistant? 
(with my mqtt explorer I can see espaltherma/sg/set and espaltherma/sg/state)

![Bildschirmfoto vom 2023-03-03 23-05-37](https://user-images.githubusercontent.com/122316058/222838588-9556f177-26bf-4e95-8209-f2f682404029.png)

Thank you very much for your help and have a nice weekend.

**Comment by javimurcia** [comment]
Check https://github.com/raomin/ESPAltherma/pull/261(also needs https://github.com/raomin/ESPAltherma/pull/261) 

It will show a select entity on home assistant that allows you to choose what SG mode to enable.

If the propossed PR works for you, please say so in the PR so we have more testers for this feature

**Comment by ap20132022** [comment]
dear @javimurcia 
thank you very much for your answer. I have just tried it on my (second) esp32, which on my desk is. This device doesn't have connect to my Heat pump. I do this step because of two reasons. First: I don't have to open my heat pump and second I want to test it before I install on my real (first) esp32.
After installing the PR, I see the change (see my screenshot). 
- I do now have the sg on another topic
- but the switch Altherma (which I use to turn on and off the heat pump - room thermostat) dissapear also
Is it done that way on purpose?
How can I use both switch Altherma and SG together?
Thanks again for your help and have a nice sunday.

**Comment by ap20132022** [comment]
it works now. The only thing I have to change is the configuration of SG_RELAY with LOW_TRIGGER. Looks very nice. Thank you very much for your work
